src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int SHIFT_BITS = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private final float loadFactor;
+    private int threshold;
+
+    public MyHashMap() {
+        this.capacity = 16;
```

Comment:
The initial capacity '16' should be defined as a constant to avoid magic numbers as per the checklist.